src/core/src/package_managers/YumPackageManager.py

```diff
@@ -326,6 +326,23 @@ def get_package_size(self, output):
         return Constants.UNKNOWN_PACKAGE_SIZE
     # endregion
 
+    # region auto OS updates
+    def disable_auto_os_update(self):
+        """ Disables auto OS updates on the machine only if they are enabled and logs the default settings the machine comes with """
+        pass
+
+    def is_image_default_patch_mode_backup_valid(self, image_default_patch_mode_backup):
+        return True
+
+    def backup_image_default_patch_mode(self):
+        """ Records the default system settings for auto OS updates within patch extension artifacts for future reference.
+        We only log the default system settings a VM comes with, any subsequent updates will not be recorded"""
+        pass
```

Comment:
Minor comment to think about: Look at this outside of the context you have as the author and me as the reviewer, if someone else reads this function name, would they have reason to believe it will silently not do what it says it will in some cases (it'd be different if it threw an exception)? The improvement possible here would be for the caller to make the determination as to whether to call it in the first place and not encapsulate that in the function itself. Alternatively the function could be named something like backup_image_default_patch_mode_if_not_exists. At your discretion, and food for thought.